docs/integrations.md

```diff
@@ -128,14 +128,13 @@ For more information on the GitHub Action see https://github.com/marketplace/act
 Include `semgrep` in your `.gitlab-ci.yml` configuration file:
 
 ```yaml
-stages:
-    - test
-test:
-    image: python
-    before_script:
-        - python -m pip install semgrep
-    script:
-        - semgrep --config https://semgrep.live/p/r2c /path/to/code
+include:
+  - template: 'Workflows/MergeRequest-Pipelines.gitlab-ci.yml'
```

Comment:
Is this needed for diff-awareness?

Reply:
Any way of configuring merge request based pipelines is okay, this seems to be the easiest. If you don't take care, it's really easy to accidentally run on pushes instead as that's the default behavior, which is why I included this in the example.